helm.go

```diff
@@ -164,8 +165,12 @@ This will not alter the charts in your workspace.
 			ArgsUsage: "[chart-name...]",
 			Action: func(c *cli.Context) {
 				minArgs(c, 1, "uninstall")
+				client := kubectl.Client
```

Comment:
@adamreese do we need the global `kubectl.Client` anymore? asking because this could also just be `client := kubectl.RealClient{}`. not necessary to change in this PR, just for my info